source/image-handler/image-handler.js

```diff
@@ -20,29 +20,24 @@ class ImageHandler {
     const edits = request.edits;
     const cropping = request.cropping;
 
-    logger.info("Image processing parameters", { cropping, edits });
-
     const hasEdits = edits !== undefined && Object.keys(edits).length > 0;
     const hasCropping = cropping !== undefined;
     if (hasEdits || hasCropping) {
-      let image;
-      let keys = [];
-      if (hasEdits) {
-        keys = Object.keys(edits);
-      }
+      let image = null;
+      const keys = Object.keys(edits);
 
       if (keys.includes("rotate") && edits.rotate === null) {
-        image = sharp(originalImage, { failOnError: false });
+        image = sharp(originalImage, {failOnError: false});
       } else {
         const metadata = await sharp(originalImage, {
           failOnError: false,
         }).metadata();
         if (metadata.orientation) {
-          image = sharp(originalImage, { failOnError: false }).withMetadata({
+          image = sharp(originalImage, {failOnError: false}).withMetadata({
```

Comment:
If you use vscode you can format everything with prettier so that this doesn't happen

Reply:
Makes it easier to spot where the real changes happen

Reply:
just did a actual formatting, it was quite inconsistent. You can also setup intellij do to stuff in certain ways. Suggest a codestyle, if you like?

Reply:
I think default prettier format should do the trick. See https://prettier.io/